appsec-kit-starter/src/main/java/com/vaadin/appsec/views/AppSecView.java

```diff
@@ -97,7 +97,17 @@ private Component buildScanNowButton() {
         scanNowButton.setDisableOnClick(true);
         scanNowButton.addClickListener(e -> {
             lastScannedLabel.setText("Scanning...");
-            AppSecService.getInstance().scanForVulnerabilities();
+            UI ui = UI.getCurrent();
+            AppSecService.getInstance().scanForVulnerabilities()
+                    .exceptionally(ex -> {
+                        ui.access(() -> {
+                            lastScannedLabel
+                                    .setText("Error scanning vulnerabilities.");
+                            scanNowButton.setEnabled(true);
+                            LOGGER.error("Enabling button");
```

Comment:
Could we log an error message about the failed manual scan and to add the exception too instead of "Enabling button"? Something similar what is in the `AppSecServiceInitListener`.

Reply:
Yeah, that log message was supposed to help me debug. It should've not been committed.